core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE.
       <!-- parent path -->
         <div class="parentPath">
           <form action="${backPath}" method="get">
-            <a href="${topPath}"><l:icon class="${icons.toNormalizedIconNameClass(icon)} icon-xlg" /></a>
+            <a href="${topPath}"><l:icon class="${icons.toNormalizedIconNameClass(icon)} icon-md" /></a>
```

Comment:
Unfortunately, that doesn't make it much better either 
![](https://i.imgur.com/zu8PIzT.png)

Reply:
I've updated the page, it's not perfect (I want to take another look at this page in the future) but I think it's cleaner now:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/43062514/155698536-45f39264-56ca-4f72-8b1f-288ec1a7d980.png">

Reply:
That looks much better <3